dart/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   meta: ^1.3.0
   stack_trace: ^1.10.0
   uuid: ^3.0.0
+  intl: ^0.17.0 # TODO remove or vendor it (its a dart.dev lib)
```

Comment:
See https://github.com/getsentry/sentry-dart/pull/1004/files#r974958847
We can either vendor this class or accept it, its a lib maintained by the Dart team, 19 months no new releases

Reply:
Since these decisions can be more 'platform' specific, I'll defer this to you and community. We've had a few dependencies so far without issues.

Reply:
Since it's not part of the standard lib, I'd say so, it's an official lib btw.
Dart does code stripping so bundle size is also not a concern.
The class that does it is quite big and complex, and obviously handles more cases.
https://stackoverflow.com/questions/62989638/convert-long-double-to-string-without-scientific-notation-dart
I'd rather use the lib then so it works on every platform without no surprises, including JS.